frontends/web/src/routes/device/manage-backups/manage-backups.jsx

```diff
@@ -25,7 +26,23 @@ import * as styles from './manage-backups.css';
 
 @translate()
 export default class ManageBackups extends Component {
+    getDevice = () => {
+        if (this.props.deviceID === null) {
+            return null;
+        }
+        return this.props.devices[this.props.deviceID] || null;
+    }
+
+    componentWillMount() {
+        if (this.getDevice() === null) {
+            route('/', true);
+        }
+    }
+
     render({ t, deviceID }, { }) {
+        if (this.getDevice() === null) {
```

Comment:
This seems quite defensive to me. Why do you think it's necessary / a good idea?

Reply:
If it is rendered while the device is not plugged in, the device backend handlers are called and return an error (because already uninitialized).

Reply:
But if `componentWillMount` `routes` the app away, will the component still be rendered? If no, when will there be "no device" in the `render` method? But nevermind…